File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Valuations/ValuationResponseDto.ts
export class ValuationResponseDto {
    idValuation: number; // El ID de la valoración generada
    idTrip: number;      // El ID del viaje
    userValuated: number;      // El ID del usuario que fue calificado
    userWhoValuated:number;
    rating: number;      // La calificación dada
    comments: string;    // Los comentarios adicionales
  
    constructor(idValuation: number, idTrip: number, userValuated: number , userWhoValuated:number, rating: number, comments: string) {
      this.idValuation = idValuation;
      this.idTrip = idTrip;
      this.userValuated = userValuated;
      this.userWhoValuated =userWhoValuated
      this.rating = rating;
      this.comments = comments;
    }
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/valuations/ValuationResponseDto.java
package ps.com.viajeros.dtos.valuations;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ValuationResponseDto {
    private Long idValuation;  // El ID de la valoración generada
    private Long idTrip;       // El ID del viaje
    private Long userValuated;       // El ID del usuario que fue calificado
    private Long userWhoValuated;
    private Long rating;       // La calificación dada
    private String comments;   // Los comentarios adicionales
}
